emcc.py

```diff
@@ -2856,18 +2856,28 @@ def modularize():
   with open(final_js, 'w') as f:
     f.write(src)
 
-    # Export using a UMD style export, or ES6 exports if selected
+    # Add ; if src doesn't contain it so we generate valid JS (otherwise acorn fails)
+    if src.rstrip()[-1] != ';':
```

Comment:
This is pretty ugly, but there was a case where it would generate:
```
var Module = { all the code here }
if (typeof AudioWorkletGlobalScope === 'function')
...
```
which would fail as it passed through acorn because of that missing ;

I guess this needs fixing where it's generated... I'll check this again tomorrow.